roles/pulp_common/tasks/main.yml

```diff
@@ -10,6 +10,9 @@
     var: pulp_install_plugins_normalized
     verbosity: 1
 
+- debug:
+    var: pulp_install_dir
+
```

Comment:
Are debug steps meant to have been left in?

Reply:
Nope. Removed them, thank you.